pkg/dns/dns.go

```diff
@@ -134,7 +133,7 @@ func (c *Client) kubeDNSMatch() (bool, error) {
 	}
 
 	if err != nil {
-		return false, fmt.Errorf("unable to get KubeDNS deployment in namesapce %q: %w", metav1.NamespaceSystem, err)
+		return false, fmt.Errorf("unable to get KubeDNS deployment in namespace %q: %w", metav1.NamespaceSystem, err)
 	}
 
 	c.logger.Info("KubeDNS match")
```

Comment:
We should consider changing this to a `Debug` log level. Same for:
- `Checking CoreDNS`
- `Checking KubeDNS`
- `CoreDNS match`

Ideally, the INFO output of the prepare command should be:

```
> DNS provider detected: KubeDNS
> Patching KubeDNS
> Restarting "coredns" pods
> Restarting "kube-dns" pods
```

You don't have to do it in this PR. But we need to fix this at some point, the output of the prepare command needs to be improved.

Reply:
You're right, can you create an issue for that?